src/features/metadata-instance-editor/Instance.js

```diff
@@ -351,10 +344,10 @@ class Instance extends React.PureComponent<Props, State> {
      * @return {Object} - react title element
      */
     getTitle(): React.Node {
-        const { cascadePolicy = {}, hasError, isCascadingPolicyApplicable, template }: Props = this.props;
+        const { cascadePolicy = {}, hasError, template }: Props = this.props;
         const isProperties = template.templateKey === TEMPLATE_CUSTOM_PROPERTIES;
 
-        const type = isCascadingPolicyApplicable && cascadePolicy.id ? 'cascade' : 'default';
```

Comment:
just to confirm, we're removing `isCascadingPolicyApplicable` because if `cascadePolicy` exists then the former must be true, correct? and thus the former check is redundant?